File: Android_Zone_Test/src/com/example/mylib_test/activity/db/Db_MainActivity.java
package com.example.mylib_test.activity.db;

import android.app.Activity;
import android.os.Bundle;
import android.view.View;
import android.view.View.OnClickListener;
import android.widget.Button;

import com.example.mylib_test.R;

import butterknife.BindView;
import butterknife.ButterKnife;

import com.zone.view.FlowLayout;

public class Db_MainActivity extends Activity implements OnClickListener {

    @BindView(R.id.db_addColumn)
    Button dbAddColumn;
    @BindView(R.id.db_update_Column)
    Button dbUpdateColumn;
    @BindView(R.id.db_queryColumn)
    Button dbQueryColumn;
    @BindView(R.id.db_create)
    Button dbCreate;
    @BindView(R.id.db_Tabledelete)
    Button dbTabledelete;
    @BindView(R.id.db_add)
    Button dbAdd;
    @BindView(R.id.db_addOrUpdateEntity)
    Button dbAddOrUpdateEntity;
    @BindView(R.id.db_updateByC)
    Button dbUpdateByC;
    @BindView(R.id.db_update)
    Button dbUpdate;
    @BindView(R.id.db_deleteAll)
    Button dbDeleteAll;
    @BindView(R.id.db_deleteByC)
    Button dbDeleteByC;
    @BindView(R.id.db_delete)
    Button dbDelete;
    @BindView(R.id.db_query)
    Button dbQuery;
    @BindView(R.id.db_queryInner)
    Button dbQueryInner;
    @BindView(R.id.db_queryByC)
    Button dbQueryByC;
    @BindView(R.id.tran_add_fail)
    Button tranAddFail;
    @BindView(R.id.tran_add_success)
    Button tranAddSuccess;
    @BindView(R.id.flowLayoutZone1)
    FlowLayout flowLayoutZone1;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.a_dbtest);
        ButterKnife.bind(this);

    }

    @Override
    public void onClick(View v) {
        onClickTable(v);
        onClickAdd(v);
        onClickQuery(v);
        onClickUpdate(v);
        onClickDelete(v);
    }

    private void onClickDelete(View v) {
        switch (v.getId()) {
            case R.id.db_delete:
                break;
            case R.id.db_deleteAll:
                break;
            case R.id.db_deleteByC:
                break;

            default:
                break;
        }
    }

    private void onClickUpdate(View v) {
        switch (v.getId()) {
            case R.id.db_update:
                break;
            case R.id.db_updateByC:
                break;

            default:
                break;
        }
    }

    private void onClickQuery(View v) {
        switch (v.getId()) {
            case R.id.db_query:
                break;
            case R.id.db_queryInner:
                break;
            case R.id.db_queryByC:
                break;

            default:
                break;
        }
    }

    private void onClickAdd(View v) {
        switch (v.getId()) {
            case R.id.db_add:
                break;
            case R.id.db_addColumn:
                break;
            case R.id.db_addOrUpdateEntity:
                break;
            case R.id.tran_add_fail:

                break;
            case R.id.tran_add_success:
                break;
            default:
                break;
        }
    }

    private void onClickTable(View v) {
        switch (v.getId()) {
            case R.id.db_create:
                break;
            case R.id.db_Tabledelete:
                break;
            case R.id.db_queryColumn:
                break;

            default:
                break;
        }
    }

}

File: Android_Zone_Test/src/com/example/mylib_test/activity/db/Db_MainActivity.kt
package com.example.mylib_test.activity.db

import android.view.View
import android.view.View.OnClickListener
import com.example.mylib_test.R

import com.zone.lib.base.controller.activity.BaseFeatureActivity

class Db_MainActivity : BaseFeatureActivity(), OnClickListener {

    override fun setContentView() {
        setContentView(R.layout.a_dbtest)
    }

    override fun initData() {
    }

    override fun setListener() {
    }

    override fun onClick(v: View?) {
        onClickTable(v)
        onClickAdd(v)
        onClickQuery(v)
        onClickUpdate(v)
        onClickDelete(v)
    }

    private fun onClickDelete(v: View?) {
        when (v?.id) {
            R.id.db_delete -> {
            }
            R.id.db_deleteAll -> {
            }
            R.id.db_deleteByC -> {
            }

            else -> {
            }
        }
    }

    private fun onClickUpdate(v: View?) {
        when (v?.id) {
            R.id.db_update -> {
            }
            R.id.db_updateByC -> {
            }

            else -> {
            }
        }
    }

    private fun onClickQuery(v: View?) {
        when (v?.id) {
            R.id.db_query -> {
            }
            R.id.db_queryInner -> {
            }
            R.id.db_queryByC -> {
            }

            else -> {
            }
        }
    }

    private fun onClickAdd(v: View?) {
        when (v?.id) {
            R.id.db_add -> {
            }
            R.id.db_addColumn -> {
            }
            R.id.db_addOrUpdateEntity -> {
            }
            R.id.tran_add_fail -> {
            }
            R.id.tran_add_success -> {
            }
            else -> {
            }
        }
    }

    private fun onClickTable(v: View?) {
        when (v?.id) {
            R.id.db_create -> {
            }
            R.id.db_Tabledelete -> {
            }
            R.id.db_queryColumn -> {
            }

            else -> {
            }
        }
    }

}
